src/Web/Rsp.IrasPortal.Web/TagHelpers/AccessibilityHelper.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.Text.RegularExpressions;
+
+namespace Rsp.IrasPortal.Web.TagHelpers;
+
+public static partial class AccessibilityHelper
+{
+    public static string BuildAriaDescribedBy(string? input)
+    {
+        if (string.IsNullOrWhiteSpace(input))
+            return string.Empty;
```

Comment:
Please always use curly braces for if statements
```suggestion
        if (string.IsNullOrWhiteSpace(input))
        {
            return string.Empty;
         }
```